src/services/registration.js

```diff
@@ -0,0 +1,32 @@
+// const regModel = require('../models/registrationExample');
+const checkName = require('../services/checkUserName');
+const checkPsw = require('../services/checkPassword');
+const dbCheck = require('../services/regCheckDB');
+const users = require('../models/userDB');
+
+const createUser = item => new Promise((resolve) => {
+  const temp = {
+    username: '',
+    password: '',
+  };
+  if (checkName.CheckUserName(item.username) && checkPsw.CheckPassword(item.password)) {
+    if (item.password === item.confirmPsw) {
```

Comment:
This is the pyramid of doom. It's hard to track which else consequence goes to which if condition. And your elses do resolving so it would be better to check for negatives.

Reply:
I destroyed the pyramid. Now it is more like a road, you go step by step.